docs/book/src/reference/versions.md

```diff
@@ -150,16 +153,20 @@ Each Cluster API contract version defines a set of rules a provider is expected
 Those rules can be in the form of CustomResourceDefinition (CRD) fields and/or expected behaviors to be implemented.
 See [provider contracts](../developer/providers/contracts/overview.md)
 
-Each Cluster API release supports only one contract version, and by convention the supported contract version matches
+Each Cluster API release supports one contract version, and by convention the supported contract version matches
 the newest API version in the same Cluster API release.
 
-| Contract Version | Status    | Supported Until                                                               |
-|------------------|-----------|-------------------------------------------------------------------------------|
-| v1beta1          | Supported | After a newer API contract will be released                                   |
-| v1alpha4         | EOL       | EOL since 2023-12-05 - v1.6.0 release date; removal planned for v1.13, Apr 26 |
-| v1alpha3         | EOL       | EOL since 2023-07-25 - v1.5.0 release date; removal planned for v1.13, Apr 26 |
+A contract version might be compatible with older contract versions; compatibility for older contract versions will 
+be dropped when the older contract version will be considered EOL.
 
-See [11919](https://github.com/kubernetes-sigs/cluster-api/issues/11919) for details about the v1alpha3/v1alpha4 removal plan.
+| Contract Version | Compatible with contract versions | Status      | Supported Until                                            |
+|------------------|-----------------------------------|-------------|------------------------------------------------------------|
+| v1beta2          | v1beta1                           | Supported   | After a newer API contract will be released                |
```

Comment:
> After a newer API contract will be released

Seems inconsistent with "at least 9 months or 3 minor releases after a newer API version will be released" in the table above. Or at least it is confusing me

Reply:
We have to keep old API versions around for 9 months or 3 minor releases + have conversions etc. etc.

For contract version it is our choice if to keep them around.
In the past old contracts were immediately removed; for v1beta1 we decided to keep it around/considering it compatible, but we are not required to do so for v1beta2 in future. 

This is why for v1beta2 I'm documenting the worst scenario (it will go away immediately)